src/libraries/System.Reflection.MetadataLoadContext/Directory.Build.props

```diff
@@ -2,5 +2,7 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <StrongNameKeyId>Open</StrongNameKeyId>
+    <PackageVersion>5.0.1</PackageVersion>
+    <AssemblyVersion>5.0.0.1</AssemblyVersion>
```

Comment:
Are you sure this one is OK?  I think WindowsDesktop includes this, not sure if they expose it.

Reply:
According to the servicing docs, it isn't part of their targeting pack: https://github.com/dotnet/runtime/blob/release/5.0/docs/project/refpack-assemblies.md.

Reply:
Not exposed in their ref-pack it seems, only consumed by the SDK.